Trove.js

```diff
@@ -0,0 +1,337 @@
+			delete item.itemID;
+			delete item.type;
+			if (thumbnailURL !== null) {
+				item.attachments.push({url: thumbnailURL, title: 'Trove thumbnail image', mimeType:'image/jpeg'});
+			}
+			item.complete();
+		});
+		translator.translate();
+	});
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://trove.nla.gov.au/version/208674371",
+		"items": [
+			{
+				"itemType": "artwork",
+				"title": "THIRROUL - Hotels - Rex Hotel",
+				"creators": [
+					{
+						"firstName": "William A. (William Alan)",
+						"lastName": "Bayley",
+						"creatorType": "author"
+					}
+				],
+				"date": "1960",
+				"abstractNote": "Thirroul Hotels REX HOTEL ca. 1956 1950-1960 Illawarra Region",
+				"libraryCatalog": "Trove",
+				"url": "http://trove.nla.gov.au/version/208674371",
+				"attachments": [
+					{
+						"title": "Trove thumbnail image",
+						"mimeType": "image/jpeg"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://trove.nla.gov.au/version/11567057",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Experiences of a meteorologist in South Australia",
+				"creators": [
+					{
+						"firstName": "Clement L. (Clement Lindley)",
+						"lastName": "Wragge",
+						"creatorType": "author"
+					}
+				],
+				"date": "1980",
+				"ISBN": "9780908065073",
+				"abstractNote": "In 14 libraries. 24 p. : ill. ; 22 cm. Wragge, Clement L. (Clement Lindley), 1852-1922. South Australia. Climate, 1883-1884. Meteorologists -- South Australia -- Biography. South Australia -- Climate -- History.",
+				"extra": "Reprinted from Good words for 1887/ edited by Donald Macleod, published: London: Isbister and Co",
+				"language": "English",
+				"libraryCatalog": "Trove",
+				"publisher": "Warradale, S.Aust. : Pioneer Books",
+				"url": "http://trove.nla.gov.au/version/11567057",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://nla.gov.au/nla.news-article70068753",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "'WRAGGE.'",
+				"creators": [],
+				"date": "07 Feb 1903",
+				"abstractNote": "We have received a copy of the above which is a journal devoted chiefly to the science of meteorology. It is owned and conducted by Mr. Clement ...",
+				"libraryCatalog": "Trove",
+				"pages": "4",
+				"place": "Victoria, Australia",
+				"publicationTitle": "Sunbury News",
+				"url": "http://nla.gov.au/nla.news-article70068753",
+				"attachments": [
+					{
+						"title": "Trove newspaper PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "'WRAGGE' - we have received a copy of the above, which is a journal devoted chiefly to the science of meteorology. It is owned and conducted by Mr. Clement Wragge."
```

Comment:
It is the full text of the short newspaper article, see http://trove.nla.gov.au/newspaper/article/70068753 (the OCR version of the article is saved into a note)